src/SME.SGP.Api/Controllers/AcompanhamentoTurmaController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SME.SGP.Api.Filtros;
+using SME.SGP.Aplicacao.Interfaces;
+using SME.SGP.Infra;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SME.SGP.Api.Controllers
+{
+    [ApiController]
+    [Route("api/v1/acompanhamento/turmas")]
+    [ValidaDto]
+    public class AcompanhamentoTurmaController : Controller
+    {
+        [HttpPost("")]
+        [ProducesResponseType(typeof(IEnumerable<AuditoriaDto>), 200)]
```

Comment:
Retorna a auditoria de 1 entidade
```suggestion
        [ProducesResponseType(typeof(AuditoriaDto), 200)]
```

Reply:
Ajustado